dot/genesis.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/ChainSafe/gossamer/dot/core/types"
 	"github.com/ChainSafe/gossamer/dot/state"
 	"github.com/ChainSafe/gossamer/lib/common"
+	dbutils "github.com/ChainSafe/gossamer/lib/common/database"
```

Comment:
I don't think it makes sense to add a database directory to `lib/common` and then to have to modify the import because we have two packages named `database`.